examples/options/src/program_options.cpp

```diff
@@ -57,17 +57,15 @@ program_options::program_options(
     }
 
     // Tell the user what's happening.
-    TRACCC_INFO("Running " << description);
+    TRACCC_INFO("Running " << description << "\n\n");
```

Comment:
I wouldn't add unnecessary newlines here, just for the reason that it adds empty logging lines.